### ISSUE TITLE
Should we link to our paper on our README?

### ISSUE THREAD

**Issue by DougBurke** [sherpa/sherpa#2136]
Should we include

Sherpa: An Open Source Python Fitting Package
https://arxiv.org/abs/2409.10400 

in https://github.com/sherpa/sherpa/blob/main/README.md ?

**Comment by hamogu** [comment]
Yes, we should. I suggest we also should also update the citation instructions in all locations where we have them (docs, CIAO pages, etc.) this this paper.
However, I would wait until it's actually published in ApJ so that we don't have to do the updates twice (first to astro-ph and then in a a few weeks from astro-ph to ApJ).

**Comment by hamogu** [comment]
See also #1233

**Comment by anetasie** [comment]
Yes, we should link to the paper! I was waiting for a more permanent ApJ Suppl reference/citation before posting it on the web site.

**Comment by DougBurke** [comment]
I just wanted to know if we wanted a PR for this release. A PR I don't have time to work on, I should say ;-)

**Comment by anetasie** [comment]
I was just looking at the astropy acknowledge page. We might do something similar. I'll try to make a PR for this.
[](https://www.astropy.org/acknowledging.html)

**Comment by DougBurke** [comment]
@anetasie - can we close this now?

**Comment by hamogu** [comment]
It seems that this has been addressed.
